requirements-dev.txt

```diff
@@ -3,6 +3,7 @@ coveralls==1.1
 flake8==3.3.0
 flake8-blind-except==0.1.1
 flake8-docstrings==1.1.0
+pydocstyle==2.0.0
```

Comment:
Can you file an issue related to the pydocstyle issue? Otherwise this looks good to me